osquery/config/config.cpp

```diff
@@ -41,12 +41,6 @@ Status Config::load() {
 
   boost::unique_lock<boost::shared_mutex> lock(rw_lock);
 
-  // Set up the active config plugin once when the config is first loaded.
-  if (!getInstance().loaded_) {
```

Comment:
Registry::setUp will now inspect registries to see if there's an active plugin. If an active plugin is used it will only setUp that plugin.